labcontrol/db/tests/test_process.py

```diff
@@ -1935,20 +1937,19 @@ def test_generate_amplicon_prep_information(self):
         tester = SequencingProcess(1)
         obs = tester.generate_prep_information()
         exp_key = 'Test Run.1'
-        exp = {exp_key: COMBINED_SAMPLES_PREP_EXAMPLE}
+        exp = {exp_key: COMBINED_SAMPLES_AMPLICON_PREP_EXAMPLE}
         self.assertEqual(len(obs), len(exp))
         self.assertEqual(obs[exp_key], exp[exp_key])
 
     def test_generate_metagenomics_prep_information(self):
-        # Shotgun run--prep not implemented
-        exp_err = "Prep file generation is not implemented for " \
-                  "Metagenomics assays."
         tester = SequencingProcess(2)
-        with self.assertRaisesRegex(ValueError, exp_err):
-            tester.generate_prep_information()
+        obs = tester.generate_prep_information()
+        exp_key = 'TestShotgunRun1'
+        exp = {exp_key: COMBINED_SAMPLES_METAGENOMICS_PREP_EXAMPLE}
+        self.assertEqual(len(obs), len(exp))
+        self.assertEqual(obs[exp_key], exp[exp_key])
```

Comment:
Would it make sense for this to be `self.assertEqual(obs, exp)`?

Reply:
Not sure where to go on this one; assuming the sample output file represents correct output, directly comparing the results of the method with established results seems effective.

The only thing I can think of is that the output of the SQL query isn't ordered, so it's perhaps possible for the row ordering to differ on an otherwise identical product.

Reply:
Within this test, is there anything else that comes down in the call to `generate_prep_information`?

Reply:
Nothing. Generate_prep_information is just a fork to go to either amplicon prep info file generation or meta genomics prep info file generation, based on internal metadata. The only thing I can see coming back in addition to the output is the counterpart to exp_key itself, the 'obs_key' if you will.